src/main/java/org/openRealmOfStars/player/SpaceRace/SpaceRaceUtility.java

```diff
@@ -116,29 +116,25 @@ public static String getRandomName(final SpaceRace race,
     StringBuilder sb = new StringBuilder();
     if (DiceGenerator.getRandom(1) == 0) {
       if (race == SpaceRace.HUMAN) {
-        if (DiceGenerator.getRandom(1) == 0) {
-          sb.append("Terran");
-        } else {
-          sb.append(race.getNameSingle());
-        }
+    	  addSecondaryName(race, sb, "Terran");
       } else if (race == SpaceRace.MECHIONS) {
-        if (DiceGenerator.getRandom(1) == 0) {
-          sb.append("Steel");
-        } else {
-          sb.append(race.getNameSingle());
-        }
+    	  addSecondaryName(race, sb, "Steel");
+      } else if (race == SpaceRace.SPORKS) {
+    	  addSecondaryName(race, sb, "Taurus");
+      }	else if (race == SpaceRace.GREYANS) {
+    	  addSecondaryName(race, sb, "Aesir");
+      } else if (race == SpaceRace.CENTAURS) {
+          addSecondaryName(race, sb, "Aries");
```

Comment:
For Centaurs I would prefer Sagittarius or maybe variation like Sagittarian. But generally I like your idea naming these by zodiacs.